packages/zendesk-adapter/src/filters/view.ts

```diff
@@ -24,7 +24,7 @@ import { applyforInstanceChangesOfType } from './utils'
 
 export const VIEW_TYPE_NAME = 'view'
 
-const valToString = (val: Value): string | string[] => (_.isArray(val) ? val.map(String) : val.toString())
+const valToString = (val: Value): string | string[] => (_.isArray(val) ? val.map(String) : val?.toString())
```

Comment:
let's also change the return value
```suggestion
const valToString = (val: Value): string | string[] | undefined => (_.isArray(val) ? val.map(String) : val?.toString())
```